config/clients/python/template/README_calling_api.mustache

```diff
@@ -0,0 +1,854 @@
+#### List Stores
```

Comment:
Can we simplify the sections here? We already go into depth on configuration options above, here we need to focus on those of this particular method - e.g. the `ListStores` one can just say:

```python
configuration = openfga_sdk.Configuration(
    scheme = os.environ.get({{appUpperCaseName}}_API_SCHEME),
    api_host = os.environ.get({{appUpperCaseName}}_API_HOST),
)

response = api_instance.list_stores(page_size=page_size, continuation_token=continuation_token)
# response = ListStoreResponse(...)
# response.stores: [{ "id": "01FQH7V8BEG3GPQW93KTRFR8JB", "name": "FGA Demo Store", "created_at": "2022-01-01T00:00:00.000Z", "updated_at": "2022-01-01T00:00:00.000Z" }]
```

Reply:
The last comment is to show users quickly what they can expect the response to be like

Reply:
When using examples, please use the ones from the other SDKs to keep them consistent

Reply:
@adriantam only this remains - if we can condense the sections in the calling api to something like the one above? What do you think?

Reply:
It's just they are repetitive and take away from what the user should be focusing on. We can keep a full example installation or configuration

Reply:
@rhamzeh : the reason why I include them is that some API requires store_id while others do not.  If we don't include the configuration, how will they know which API requires store_id?

Reply:
As discussed offline, I have simplified the calling api section